lib/cloudregister/registerutils.py

```diff
@@ -2313,14 +2313,17 @@ def is_suma_instance():
     num_matches = 0
     suma_prods = [
         '/etc/products.d/suse-manager-server.prod',
-        '/etc/products.d/multi-linux-manager-server.prod',
-        '/etc/products.d/sle-micro.prod'
+        '/etc/products.d/multi-linux-manager-server.prod', # SUMA 5.1
+        '/etc/products.d/sle-micro.prod',
+        '/etc/products.d/sl-micro.prod' # Micro 6.1 and 6.2
     ]
     for product in products:
         if product.lower() in suma_prods:
             num_matches += 1
 
-    return num_matches == len(suma_prods) - 1
+    # only one Micro + SUMA product possible at the same time for
+    # a SUMA instance
+    return num_matches == 2
```

Comment:
we are not checking if there is one Micro and one SUMA prod, I understand we do not need to do that here, that would be a different problem